### ISSUE TITLE
It's not settings the custom config

### ISSUE THREAD

**Issue by danestves** [tw-in-js/typography#3]
I have this config:
![image](https://user-images.githubusercontent.com/31737273/111844430-6a70f300-88d9-11eb-93d7-78663ebfa53e.png)

But the browser render this:
![image](https://user-images.githubusercontent.com/31737273/111844453-73fa5b00-88d9-11eb-808f-69614a9dcdf6.png)

**Comment by sastan** [comment]
Yeah. The theme values are not respected. I need to re-factor this plugin.

**Comment by patheticGeek** [comment]
> Yeah. The theme values are not respected. I need to re-factor this plugin.

Hey, I need to use this in a project. Any status on this? I'd love to give this a try if someone's not on this yet.
